stm32-modules/include/thermocycler-refresh/systemwide.h

```diff
@@ -7,6 +7,9 @@
 /* size of array for setting serial number */
 #define SYSTEM_WIDE_SERIAL_NUMBER_LENGTH 24
 
+/* Uncomment this to include aynchronous error messages */
+//#define SYSTEM_ALLOW_ASYNC_ERRORS
```

Comment:
This might be better off as a build system define done by cmake rather than a change in source